x-pack/plugins/apm/public/components/routing/templates/no_data_config.ts

```diff
@@ -12,22 +12,30 @@ export function getNoDataConfig({
   docsLink,
   basePath,
   hasData,
+  hasApmIntegrations,
 }: {
   docsLink: string;
   basePath?: string;
   hasData?: boolean;
+  hasApmIntegrations?: boolean;
 }): KibanaPageTemplateProps['noDataConfig'] {
   // Returns no data config when there is no historical data
   if (hasData === false) {
+    const title = hasApmIntegrations
+      ? i18n.translate('xpack.apm.noDataConfig.addDataButtonLabel', {
+          defaultMessage: 'Add data',
+        })
+      : i18n.translate('xpack.apm.noDataConfig.addApmIntegrationButtonLabel', {
+          defaultMessage: 'Add the APM integration',
+        });
```

Comment:
What about the loading state? I don't we should show either of these options until we know which one is the right one

Reply:
Makes sense, so what do you think it should be displayed instead?